logging/src/commonMain/kotlin/social/androiddev/common/logging/LoggingModule.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is part of Dodo.
+ *
+ * Dodo is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * Dodo is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Dodo.
+ * If not, see <https://www.gnu.org/licenses/>.
+ */
+package social.androiddev.common.logging
+
+import io.github.aakira.napier.DebugAntilog
+import io.github.aakira.napier.Napier
+import org.koin.dsl.module
+
+val loggingModule = module {
+    // todo: provide different antilog for release
```

Comment:
@crocsandcoffee How do we handle TODOs like this? create a feature request? Or just in code (im fine with)

Reply:
So for TODOs that I know I am going to fix right away, I don't bother making an issue. For ones that will be revisited later, create a issue